.github/workflows/kitctl-test.yml

```diff
@@ -0,0 +1,218 @@
+name: This workflow automates the build, testing, and bootstrapping of a test environment on Kubernetes using KIT CLI
```

Comment:
This is too long for a GH action name. See how its GH ci badge renders in the README.
Maybe just call it `kit-e2e`.
Let's fix this in a followup PR